Dockerfile

```diff
@@ -124,7 +124,7 @@ RUN apt-get update \
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o rustup.sh \
   && sh rustup.sh -y --target 1.68.2-aarch64-unknown-linux-gnu 1.68.2-x86_64-unknown-linux-gnu
 
-RUN "$HOME/.cargo/bin/cargo" install sccache
+RUN "$HOME/.cargo/bin/cargo" install sccache --version 0.3.3
```

Comment:
This is mentioned here:
https://rdxworks.slack.com/archives/C01M90Y2448/p1685407332681409